lib/reporters/default.js

```diff
@@ -370,8 +370,11 @@ export default class Reporter {
 			}
 
 			this.lineWriter.writeLine(`${testsInFile.size} tests were pending in ${this.relativeFile(file)}\n`);
+			const testTitleToLogs = evt.pendingTestsLogReference.get(file) ?? new Map();
 			for (const title of testsInFile) {
+				const logs = testTitleToLogs.get(title) ?? [];
```

Comment:
`writeLogs()` can handle falsey `logs` values. So with optional chaining and the above we can do:

```suggestion
				const logs = testTitleToLogs?.get(title);
```